### PR TITLE
Molteno entropies

### PR DESCRIPTION
**Pull request by apbraun**
The interface changes to the `genentropy` function were included into `molteno_dim`.

### CODE REVIEW

File: src/dimensions/molteno.jl [hunk 1]
```diff
@@ -23,9 +23,9 @@ estimate.
 
 [^Molteno1993]: Molteno, T. C. A., [Fast O(N) box-counting algorithm for estimating dimensions. Phys. Rev. E 48, R3263(R) (1993)](https://doi.org/10.1103/PhysRevE.48.R3263)
 """
-function molteno_dim(α, data, k0 = 10; base = Base.MathConstants.e)
+function molteno_dim(α, data, k0 = 10; base = ℯ)
```
Comment (Datseris) [inline src/dimensions/molteno.jl:26]: I've changed the entropy signature of `generalized_entropy` to have `data` as first argument and `α` as keyword. Can you change this here as well? `α, k0` should be keyword arguments instead.

Comment (apbraun) [reply src/dimensions/molteno.jl:26]: sure.

Comment (apbraun) [reply src/dimensions/molteno.jl]: I fucked up the Commit message but `α` and `k0` are keyword arguments now.

Comment (apbraun) [reply src/dimensions/molteno.jl]: Should the `q` of the q-order correlation sum be also renamed to `α` to be identical to the naming in the paper?

**Pull request merged**